lib/plugins/aws/lib/naming.js

```diff
@@ -49,6 +49,22 @@ module.exports = {
     return `${this.provider.serverless.service.service}-${this.provider.getStage()}`;
   },
 
+  getServiceArtifactName() {
+    return `${this.provider.serverless.service.service}.zip`;
```

Comment:
Thanks for pointing that out. I made sure `package.artifact` is not dropped in this change to keep it backward compatible.